source/components/handling-events.md

```diff
@@ -0,0 +1,129 @@
+Instead of having to register event listeners on elements you'd like to
+respond to, simply implement the name of the event you want to respond to
+as a method on your component.
```

Comment:
I'd suggest a more positive introduction, like "You can respond to user events on your component like double-clicking, hovering, and key presses through event handlers." Starting out by contrasting Ember to another approach feels negative and assumes the reader comes from a particular background.

I think it would better to introduce event handling as a solution to a problem, rather than as an alternative to